src/status_im/contexts/wallet/data_store.cljs

```diff
@@ -165,10 +164,7 @@
 
 (defn- add-keys-to-saved-address
   [saved-address]
-  (-> saved-address
-      (assoc :network-preferences-names
-             (network-utils/network-preference-prefix->network-names (:chain-short-names saved-address)))
-      (assoc :ens? (not (string/blank? (:ens saved-address))))))
+  (assoc saved-address :ens? (not (string/blank? (:ens saved-address)))))
```

Comment:
Removed the `network-preferences-names` key which was added by client to existing saved address data for easy display in UI. Since, we removed multi chain support. It's not needed.